docs/src/layouts/AppFooter.tsx

```diff
@@ -113,7 +113,7 @@ export default function AppFooter() {
         }}
       >
         <Typography color="text.secondary" variant="body2">
-          Copyright © {new Date().getFullYear()} Material-UI SAS.
+          Copyright © {new Date().getFullYear()} MUI SAS.
```

Comment:
The name of the company didn't change, yet, I doubt we should update this.

```suggestion
          Copyright © {new Date().getFullYear()} Material-UI SAS.
```